src/marine_eov_broker/MarineRiBroker.py

```diff
@@ -394,51 +401,59 @@ def submit_request(self,
 
         return response
 
-    def submit_sparql_query(self,
-                             query,
-                             endpoint,
-                             eovs,
-                             query_start_date,
-                             query_end_date,
-                             query_min_lon,
-                             query_min_lat,
-                             query_max_lon,
-                             query_max_lat,
-                             output_format,
-                             linked_var,
-                             linked_dataset,) -> list:
-        wrapper = SPARQLWrapper(endpoint)
-        wrapper.setReturnFormat(JSON)
-        wrapper.setQuery(query)
-        results = wrapper.queryAndConvert()
+    def submit_sparql_query(
```

Comment:
so after all no `submit_sparql_named_query` variant that also abuses the templating 
(further cleaning the ipynb code not worth it?)

Reply:
Added now.
Not sure if ipynb cleaning is needed though.
The best way to implement `submit_sparql_named_query` would also imply a refactor in most of the function the `submit_query` uses